sphinxcontrib/aafig.py

```diff
@@ -96,12 +98,22 @@ def run(self):
         return [image_node]
 
 
+def warn(app, message):
```

Comment:
This is just a minor suggestion, you are welcome to leave it as is, but any reason not to take the `builder` instead of the `app` here?